### ISSUE TITLE
Configuration not working...

### ISSUE THREAD

**Issue by im1dermike** [iominh/ng-scrollbars#23]
How do you non-globally configure settings?  This isn't working:

```
<div ng-scrollbars ng-scrollbars-config="scrollbarConfig"></div>

$scope.scrollbarConfig = { axis: 'x' };
```

**Comment by iominh** [comment]
Ah hmm, I'll take a closer look this weekend or when I get a chance, but does the following demo work for you? http://iominh.github.io/ng-scrollbars/demo1.html

**Comment by im1dermike** [comment]
Not sure how, but all of the sudden it started working.

**Comment by iominh** [comment]
Ah, gremlins perhaps. Thanks though!

**Comment by im1dermike** [comment]
Gnomes ;)

**Comment by silverrajat88** [comment]
iominh, I am facing the same issue. config is not working for me.
and I have done everything by the book. are more people facing this problem?

**Comment by iominh** [comment]
Hi @silverrajat88! I replied in the other ticket https://github.com/iominh/ng-scrollbars/issues/26. Version 0.0.9 will be rolling out in the near future, which should hopefully fix this issue

**Comment by silverrajat88** [comment]
@iominh Thanks for your help.

**Comment by iominh** [comment]
Sure thing! I just released 0.0.9, so if you bower update or npm install you should hopefully see the new version

**Comment by silverrajat88** [comment]
sure thing i will check it out :)
